pkgs/applications/misc/msglint/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "msglint";
+  version = "1.04";
+
+  sourceRoot = ".";
```

Comment:
Why is this needed?

Reply:
The tarball does not contain a directory, but extracts the files in the current working directory. Thus, one manually has to adjust the `sourceRoot`.